docs/Concepts/Node-Keys.md

```diff
@@ -43,8 +43,8 @@ Specified by the [`--p2p-host`](../Reference/CLI/CLI-Syntax.md#p2p-host) and
 The enode is displayed when starting a Besu node and can be obtained using the [`net_enode`](../Reference/API-Methods.md#net_enode) 
 JSON-RPC API method. 
 
-If [UPnP](../HowTo/Find-and-Connect/Using-UPnP.md) is enabled, the enode advertised to other nodes during discovery is the 
-external IP address and port. 
+The enode advertised to other nodes during discovery is the external IP address and port as defined 
```

Comment:
Nitpicking here. Comma between "... IP address and port" and "as defined ..."?

Reply:
Fixed